### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dkraeker**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Minecraft Viewer Boilerplate](https://ngokevin.github.io/aframe-minecraft-viewer-boilerplate/) - Boilerplate for viewing [Minecraft](http://minecraft.net/)-[esque](http://www.realtimerendering.com/erich/minecraft/public/mineways/) scenes [[Code]](https://github.com/ngokevin/aframe-minecraft-viewer-boilerplate/)
 - [Aztec Pong](http://meta-meta.github.io/aztec-pong/) - Play Pong floating atop an Aztec monument [[Code]](https://github.com/meta-meta/aztec-pong/)
 - [Hello, Pizza!](http://carpedmt.github.io/) - If a pizza spins in a forest and no one is around to eat it, does it make a sound? [[Cocde]](https://github.com/carpedmt/carpedmt.github.io/)
+[Ecommerce Gallery](http://vr.gadgetplus.ca/) (by [@drawvr]) Simple picture gallery
```
Comment (ngokevin) [inline README.md]: need a dash before link and before description

File: README.md [hunk 2]
```diff
@@ -133,6 +133,7 @@ Use React with A-Frame. Share your A-Frame React Components!
 - [Minecraft Viewer Boilerplate](https://ngokevin.github.io/aframe-minecraft-viewer-boilerplate/) - Boilerplate for viewing [Minecraft](http://minecraft.net/)-[esque](http://www.realtimerendering.com/erich/minecraft/public/mineways/) scenes [[Code]](https://github.com/ngokevin/aframe-minecraft-viewer-boilerplate/)
 - [Aztec Pong](http://meta-meta.github.io/aztec-pong/) - Play Pong floating atop an Aztec monument [[Code]](https://github.com/meta-meta/aztec-pong/)
 - [Hello, Pizza!](http://carpedmt.github.io/) - If a pizza spins in a forest and no one is around to eat it, does it make a sound? [[Cocde]](https://github.com/carpedmt/carpedmt.github.io/)
+[Ecommerce Gallery](http://vr.gadgetplus.ca/) (by [@drawvr]) - Simple picture gallery
```
Comment (cvan) [inline README.md]: add a `-`

**Comment by cvan** [comment]
@dkraeker can you add the hyphen and rebase? thanks!

**Comment by ngokevin** [comment]
needs rebase but i added for you

**Pull request closed without merging**